components/_global-navigation.scss

```diff
@@ -171,7 +170,7 @@ $notification-counter-size: 18px;
 		// has `width` set in `%` and is inside `flexbox`.
 		min-width: 0;
 		padding: 3px 4px;
-		width: 100%;
+		width: 0;
```

Comment:
I first wanted to check if it actually works. And it seems it does. Thanks for pointing this out. I will fix it. 👍